code/clue.js

```diff
@@ -46,14 +192,72 @@ const pickKiller = () => {
   // This will randomly select a killer from the suspects. And add that to the mystery object.
   mystery.killer = randomSelector(suspects)
 
+
   // This will change the background color of the card to the one connected to the chosen killer and show the full name of the killer. Feel free to add more things to show about the killer.
   document.getElementById('killerCard').style.background = mystery.killer.color
   document.getElementById(
     'killerName'
   ).innerHTML = `${mystery.killer.firstName} ${mystery.killer.lastName}`
+  document.getElementById(
+    'killerDescription'
+  ).innerHTML = `${mystery.killer.description}`
+  document.getElementById(
+    'killerAge'
+  ).innerHTML = `${mystery.killer.age}`
+  document.getElementById(
+    'killerOccupation'
+  ).innerHTML = `${mystery.killer.occupation}`
+  const theKillerImage = document.getElementById(
+    'killerImage');
+  theKillerImage.src = mystery.killer.image;
 }
 
 // CREATE FUNCTIONS pickWeapon and pickRoom in a similar way.
 
+//FUNCTION FOR pickWeapon:
+
+const pickWeapon = () => {
+  mystery.weapon = randomSelector(weapons)
+  document.getElementById('weaponCard').style.background = 'darkred';
+  document.getElementById(
+    'weaponName'
+  ).innerHTML = `${mystery.weapon.name} with a weight of ${mystery.weapon.weight}`
+  const theWeaponImage = document.getElementById(
+    'weaponImage');
+  theWeaponImage.src = mystery.weapon.image;
+}
+
+//FUNCTION FOR pickRoom:
+
+
+const pickRoom = () => {
+  mystery.room = randomSelector(rooms)
+  document.getElementById('roomCard').style.background = 'darkslategray';
```

Comment:
Interesting color! Never heard of `darkslategray ` before! :D

Reply:
It's pretty cool, I found it here among with other cool ones: https://htmlcolorcodes.com/color-names/

Reply:
Thank you for sharing!